setup.py

```diff
@@ -30,23 +30,25 @@
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
         "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.6",
         "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
         "Topic :: Software Development :: Quality Assurance"
     ],
     keywords=[
         "twisted", "checker", "compliance", "pep8"
     ],
     install_requires=[
```

Comment:
The problem is that bumping the version explodes everything, because pylint has no such thing as API stability ;)

This PR is actually "Upgrade the PyLint to a version that actually works on Python 3", with some "make things work on Python 3" as well :)